src/phylo_flags.hpp

```diff
@@ -0,0 +1,254 @@
+// Copyright 2019-2021 bito project contributors.
+// bito is free software under the GPLv3; see LICENSE file for details.
+//
+// PhyloFlags are used for adding optional arguments to functions that are specified by
+// the user. Additionally, it contains the keys used for accessing members of the
+// GradientMap returned by PhyloGradients.
+//
+
+#ifndef SRC_PHYLO_FLAGS_HPP_
```

Comment:
after you rebase this can become a #pragma once as in the rest of the modern code.

Reply:
Fixed.